config/core/deployments/sources-controller.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(n3wscott): DELETE THIS FILE AFTER v0.13 CUTS
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sources-controller
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: sources-controller
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: sources-controller
+    spec:
+      containers:
+      - name: controller
+        # This is the Go import path for the binary that is containerized
+        # and substituted here.
+        image: knative.dev/eventing/cmd/sources_controller
```

Comment:
I'm presuming this won't be a problem because no pods will ever be created, so the invalid image url will never be evaluated.

Reply:
similar:
* https://github.com/matzew/eventing-sources/blob/cd6e0e83517dbfaa44bdfe7422cbc13300764ef4/kafka/source/config/500-controller.yaml#L64-L76

or 
* https://github.com/knative/eventing/commit/e88640ba0072e2ccc1c05a95c086b8d7f1591d4b#diff-a478995e243b5d4f0133ed5807fef38e